js/main.js

```diff
@@ -125,37 +127,94 @@ const addUpdateModal = function (isAdd, empId) { // Handle modal on update or ad
     const modalBackground = document.getElementById('modal-background');
     const modalContent = modalBackground.querySelector('#addUpdate-emp');
     const modalHeading = modalContent.querySelector('#modal-heading');
+    const skillDropdown = modalContent.querySelector('#addUpdate-skill-dropdown');
     const modalDropdown = modalContent.querySelector('.dropdown-element');
     const formSubmit = modalContent.querySelector('#form-submit');
     const formCancel = modalContent.querySelector('#form-cancel');
 
     modalHeading.innerText = (isAdd) ? 'Add Employee Details' : 'Update Employee Details';
     formSubmit.value = (isAdd) ? 'SAVE' : 'UPDATE';
 
-    if (!isAdd) {
-        fillUpdateClearForm(false, empId);
-    }
-
     modalBackground.classList.remove('display-none');
     modalContent.classList.remove('display-none');
 
+    skillDropdown.onclick = (event) => {
+        if (event.target.dataset.skillid.startsWith('id-')) {
+            const skillInputTag = document.getElementById('emp-skill');
+            const skillName = event.target.innerText;
+            let skillInput = document.getElementById('emp-skill').value.split(', ');
+
+            skillInput.push(skillName);
+
+            if (skillInput[0] == '') {
```

Comment:
@aromal-qburst If you are trying to remove whitespace, use trim instead

Reply:
Done